### PR TITLE
Bug 1690181 - Rename gecko-dev.tar to gecko.tar

### PR DESCRIPTION
**Pull request by staktrace**
All the other repos have their git repo in `<foo>.tar` and
blame repo in `<foo>-blame.tar`. gecko-dev is unique in using
`gecko-dev.tar` and `gecko-blame.tar`. This patch brings gecko
in line with the other repos, by changing the git tarball to
be `gecko.tar`. Note that the folder inside is still `gecko-dev`;
eventually it will be renamed to `git` to match other
repositories but that'll happen when I'm more in the mood to
manually download and re-upload a 10G tarball.

### CODE REVIEW

**Comment by staktrace** [comment]
I ran `aws s3 cp s3://searchfox.repositories/gecko-dev.tar s3://searchfox.repositories/gecko.tar --acl public-read` to copy over the gecko-dev.tar to gecko.tar, and am running a `kats` channel indexer with this change to confirm it doesn't break anything. Assuming it's good, this PR should be ok to merge once approved, but it would be advisable to re-copy over gecko-dev.tar to gecko.tar around when the PR is merged, to ensure it's at the latest version. And then some time later we can delete `gecko-dev.tar`.

**Comment by staktrace** [comment]
Ditto with `aws s3 cp s3://searchfox.repositories/nss-git.tar s3://searchfox.repositories/nss.tar --acl public-read`

**Review by asutherland — APPROVED** [review]
Argh, sorry I didn't get to this before/on the weekend; I expect you also have been having fun with snow too!  I'm going to leave this to you to push buttons on if that's okay, but I can do it too if desired.

**Comment by staktrace** [comment]
> Argh, sorry I didn't get to this before/on the weekend; I expect you also have been having fun with snow too! I'm going to leave this to you to push buttons on if that's okay, but I can do it too if desired.

No worries. The snow was fun, but would have been better without the excessive wind :)

**Pull request merged**
